apps/api/src/app/bridge/dtos/create-bridge-request.dto.ts

```diff
@@ -1,10 +1,15 @@
-import { IsOptional, IsString } from 'class-validator';
+import { IsEnum, IsOptional, IsString } from 'class-validator';
 import { ICreateBridges, IWorkflowDefine } from '../usecases/sync';
+import { WorkflowOriginEnum } from '@novu/shared';
 
 export class CreateBridgeRequestDto implements ICreateBridges {
   workflows: IWorkflowDefine[];
 
   @IsOptional()
   @IsString()
   bridgeUrl: string;
+
+  @IsEnum(WorkflowOriginEnum)
+  @IsOptional()
```

Comment:
To be explored:
@tatarco  says it shouldn't be here, should be in the IWorkflowDefine